lib/api/core/statistics.js

```diff
@@ -145,6 +145,55 @@ module.exports = function (kuzzle) {
   };
 
   /**
+   * Gets the lasts saved statistics frame from a date
+   *
+   * @param requestObject
+   * @returns {Promise} promise resolved as a ResponseObject
+   */
+  this.getStats = function(requestObject) {
+    var
+      deferred = q.defer(),
+      response = {
+        statistics: {}
+      },
+      frames,
+      currentDate,
+      since;
+
+    currentDate = new Date().getTime();
+    since = requestObject.data.body.since;
+    if (since > currentDate) {
+      deferred.resolve(new ResponseObject(requestObject, response));
```

Comment:
I don't think it is an incorrect argument, we only assume there is no statistics in the future therefore it returns an empty array and do not throw an error as if it was a malformatted date